src/disco/containers/DiscoPane.js

```diff
@@ -55,7 +59,11 @@ export class DiscoPaneBase extends React.Component {
   }
 
   showVideo = (e) => {
-    const { _tracking } = this.props;
+    const { _tracking, _video } = this.props;
+    if (_video) {
+      this.video = _video;
```

Comment:
Re-assigning `this.video` makes me nervous because refs are kind of magical. I think we need to get this working:
````js
const { _video } = this.props._video || this.video;
````